ofdb-entities/src/email.rs

```diff
@@ -1,62 +1,85 @@
 use std::{borrow::Borrow, fmt, ops::Deref, str::FromStr};
+use thiserror::Error;
 
-// TODO: rename to EmailAddress
 #[derive(Debug, Clone, Eq, PartialEq, Ord, PartialOrd)]
-pub struct Email(String);
+pub struct EmailAddress(String);
 
-impl AsRef<String> for Email {
+impl EmailAddress {
+    pub const fn from_string_unchecked(address: String) -> Self {
```

Comment:
`new_unchecked()`

Providing `new()` and `into_inner()` for newtypes seems to be the idiomatic way.